src/content/posts/2022/bootstrap-4-6-2.md

```diff
@@ -0,0 +1,37 @@
+---
+author: mdo
+date: "2022-07-14T21:30:00Z"
+title: Bootstrap v4.6.2
+video: RDTzFnYcIqj6I
+video_start: 135
+keywords:
+  - bootstrap
+  - release
+---
+
+After several months, we've finally shipped Bootstrap v4.6.2, one of our last releases for the v4. It's a bit of a maintenance patch featuring all bug fixes, dependency updates, and some docs updates.
+
+Read on for the highlights or [head to the v4.6.x docs](https://getbootstrap.com/docs/4.6/) to see the latest in action.
+
+## What's changed
+
+There are two big highlights in v4.6.2:
+
+- First, we've added an example to our Collapse plugin docs to show how to use horizontal collapsing. This has long been possible via our JS, but we never had an official class to utitlize it.
+
+- Second, we've replaced the deprecated `color-adjust` with `print-color-adjust` as part of the Autoprefixer v10.4.6 issues. This should quiet the issues folks have seen from that dependency change.
```

Comment:
Not sure if we should clarify that this only fixes the issue for people building the CSS themselves. If one is using the dist CSS file to build their CSS further, they will still see the warning.

I hit this with some Angular projects where I use the dist CSS instead of the Sass files.